packages/laverna/ava.config.js

```diff
@@ -0,0 +1,13 @@
+// if `mock` is not present in the `node:test` package, `laverna.spec.js` should be skipped
+
+const config = {
+  files: ['test/cli.spec.js'],
+}
+
+try {
+  if (require('node:test').mock) {
+    config.files.push('test/laverna.spec.js')
+  }
```

Comment:
I like this solution. Would be nice to make this more explicit.
```suggestion
  } else {
    console.error('[SKIP] Laverna tests skipped - too old Node.js version')
  }
```